src/otx/core/engine/test.py

```diff
@@ -33,7 +34,7 @@ def test(cfg: TrainConfig) -> tuple[Trainer, dict[str, Any]]:
     trainer: Trainer = hydra.utils.instantiate(cfg.trainer)
 
     log.info("Starting testing!")
-    if cfg.checkpoint is None:
+    if not isinstance(model.model.model, Model) and cfg.checkpoint is None:
```

Comment:
Why do we need this change?

Reply:
Because Openvino model runs without any checkpoints, it is a final representation of a model. So, with this condition we couldn't run "otx test" without checkpoint

Reply:
For a new comer who sees this first time, it is really hard to understand its underlying behavior. Can we enhance this code clearer? We may utilize `cfg.checkpoint` to load the OV IR file. Even if it's not exactly this idea, I think it should be improved.

Reply:
I see two options here:
1) instead of model_name we can use checkpoint to load IR model
2) remove raising error in case when checkpoint is None. In some cases even for PyTorch models we may want to use pure model with pre-trained weights to check the score on dataset

Reply:
Let's remove this check

Reply:
1st option seems to raise problems inside lightning, OV model can't be loaded in this way

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2748/commits/fa36076ef72b7ad990a68b70db33ed4eb007a889)